### ISSUE TITLE
.finally is not a function

### ISSUE THREAD

**Issue by goldo** [knee-cola/jest-mock-axios#35]
Hi, i'm getting this error in my test:
```
      Error: Uncaught [TypeError: _api.default.search(...).then(...).finally is not a function]
```

So I tried to do a simple: 
```
import axios from 'jest-mock-axios';

console.log(axios.post().finally);
```
and got `undefined`

version: `3.0.0`

**Comment by kingjan1999** [comment]
This is because our promise library (jest-mock-promise) doesn't support `finally`. There's an open merge request (knee-cola/jest-mock-promise#6) but I doubt it will get merged soon. 

However, as we probably have to replace the promise library to fix #17, this issue should be fixed as well soon.

**Comment by kingjan1999** [comment]
Fixed with 3.1.0 (just published)
